### PR TITLE
README.rst: Add link to docs

### LINKED ISSUE #2906: README: Documentation link
I didn't find a link to the documentation after glancing over the readme. This maybe should be improved.

### PR DESCRIPTION
**Pull request by achilleas-k**
This PR adds a link to the full documentation (coala.readthedocs.io) to the row of external links in the README.

Closes #2906.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,9 +30,9 @@ To see what coala can do for your language, run:
 
 .. Start ignoring LineLengthBear
 
-======================================= ================================================= ====================================================== =========================================================
-`Official Website <http://coala.io/>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
-======================================= ================================================= ====================================================== =========================================================
+======================================= ===================================================== ================================================= ====================================================== =========================================================
```
Comment (Adrianzatreanu) [inline README.rst]: it appears that this change is unrelated to this commit :D i dont think it should be changed here

File: README.rst [hunk 2]
```diff
@@ -30,9 +30,9 @@ To see what coala can do for your language, run:
...
+`Official Website <http://coala.io/>`__ `Full Documentation <https://coala.readthedocs.io>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
+======================================= ===================================================== ================================================= ====================================================== =========================================================
```
Comment (Adrianzatreanu) [inline README.rst]: same with this one :)

File: README.rst [hunk 3]
```diff
@@ -30,9 +30,9 @@ To see what coala can do for your language, run:
 
 .. Start ignoring LineLengthBear
 
-======================================= ================================================= ====================================================== =========================================================
-`Official Website <http://coala.io/>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
-======================================= ================================================= ====================================================== =========================================================
+======================================= ===================================================== ================================================= ====================================================== =========================================================
+`Official Website <http://coala.io/>`__ `Full Documentation <https://coala.readthedocs.io>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
```
Comment (Adrianzatreanu) [inline README.rst]: "Full" is too much. just "Documentation" is fine :D

File: README.rst (hunk 1 above)
Comment (achilleas-k) [inline README.rst]: In the rst source, each link is covered (above and below) by a sequence of `=`, I suspect for aesthetic reasons. This change adds an extra block to accommodate the new link.

File: README.rst (hunk 2 above)
Comment (achilleas-k) [inline README.rst]: Same reason as above.

**Comment by adtac** [comment]
ack https://github.com/coala/coala/pull/2909/commits/cdaa239cc8e96e022f3b4e144e16a8a9303dcadd

**Comment by adtac** [comment]
@rultor merge

**Comment by adtac** [comment]
Hey @achilleas-k , this is your first contribution, right? Congrats and welcome to the coalaians! :)

You can help us a lot by filling out http://coala.io/newform!

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

See also https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer btw.
